### PR TITLE
kie-server: upgrade to WildFly 10/EAP 7 + Tomcat 8

### PR DESCRIPTION
**Pull request by psiroky**
* remove old configs for WildFly 8, EAP 6
   and Tomcat 7

 * adjust profile names to synch with the kie-wb-smoke-tests
   (e.g. tomcat8x -> tomcat8; eap7x -> eap7)

 * remove ancient jbossas71x profile

### CODE REVIEW

File: kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/META-INF/kie-server-jms.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <jms-queue name="KIE.SERVER.RESPONSE">
+        <entry name="queue/KIE.SERVER.RESPONSE" />
+        <entry name="java:jboss/exported/jms/queue/KIE.SERVER.RESPONSE" />
+      </jms-queue>
+
+      <!-- Kie Server EXECUTOR queue -->
+      <jms-queue name="KIE.SERVER.EXECUTOR">
+        <entry name="queue/KIE.SERVER.EXECUTOR" />
+      </jms-queue>
+
```
Comment (mswiderski) [inline kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/META-INF/kie-server-jms.xml]: could we include the commented part for easier enablement of external signals over JMS - jBPM specific: https://github.com/droolsjbpm/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-wars/kie-server/src/main/shared-ee6-ee7-resources/META-INF/kie-server-jms.xml#L23-L30

**Comment by mswiderski** [comment]
+1 with just small comment to include KIE.SERVER.SIGNAL (disabled) queue definition

File: kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/META-INF/kie-server-jms.xml (hunk 1 above)
Comment (psiroky) [inline kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/META-INF/kie-server-jms.xml]: Yes, will do that (I took old copy of the file which did not yet had the commented part I believe). Thanks!

Comment (psiroky) [inline kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/META-INF/kie-server-jms.xml]: Added, see below.

**Comment by psiroky** [comment]
@sutaakar could you please take a look as well? (since this is also tests + cargo related)

**Comment by sutaakar** [comment]
+1

**Comment by psiroky** [comment]
jenkins retest this (wrong job config caused that the kie-server tests weren't executed :/)

**Pull request merged**